receiver/redisreceiver/metadata.yaml

```
@@ -103,7 +104,7 @@ metrics:
   redis.clients.connected:
     enabled: true
     description: Number of client connections (excluding connections from replicas)
-    unit:
+    unit: "{connections}"
```

```suggestion
    unit: "{clients}"
```
Looks like we're counting clients here (with a filter).